src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
+    private static final int RIGHT_TRIANGLE_NUMBER = 2;
+    private int sid;
```

Comment:
The variable name 'sid' is not descriptive. Consider renaming it to something more meaningful, like 'sideLength', to enhance code readability.